corehq/apps/domain/migrations/0008_use_livequery.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.2.13 on 2020-09-24 17:53
+
+from django.db import migrations
+
+from couchdbkit import ResourceNotFound
+
+from corehq.apps.domain.models import Domain
+from corehq.util.django_migrations import skip_on_fresh_install
+from toggle.models import generate_toggle_id, Toggle
+
+
+@skip_on_fresh_install
+def _set_use_livequery(apps, schema_editor):
+    toggle_id = generate_toggle_id('livequery_sync')
+    try:
+        toggle_doc = Toggle.get_db().get(toggle_id)
+    except ResourceNotFound:
+        # Flag isn't enabled for anyone on this server
+        return
+    for user in toggle_doc.get('enabled_users', []):
+        domain_obj = Domain.get_by_name(user.split('domain:')[1])
```

Comment:
Thinking out loud here, wondering if there's any chance that `user` does not start with (or contain) `domain:`? If it doesn't this will produce an error. Would it make sense to code this a bit more defensively so it just ignores any value like that? Something like
```suggestion
        if not user.startswith('domain:'):
            log.warning("unexpected livequery_sync user: %s", user)
            continue
        domain_obj = Domain.get_by_name(user[len('domain:'):])
```
also will need to create a logger near the top of this file:
```py
import logging

log = logging.getLogger(__name__)
```

Reply:
It depends on the allowable namespaces for the toggle.  In this case, it's only `NAMESPACE_DOMAIN`, so we should be good unless someone did something real wacky.

FWIW I think this can be simplified a bit with
```python
LIVEQUERY_SYNC.get_enabled_domains()
```

Reply:
oh nevermind, I see the toggle is also removed in this PR.  Carry on.